packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -75,6 +75,11 @@ export const messages = {
 
   force_source_deploy_text: 'SFDX: Deploy Source to Org',
   force_source_retrieve_text: 'SFDX: Retrieve Source from Org',
+  force_source_delete_text: 'SFDX: Delete from Project and Org',
+  force_source_delete_manifest_unsupported_message:
+    'SFDX: Delete from Project and Org is not supported for manifest files. Select a source file or directory to delete',
+  force_source_delete_confirmation_message:
+    'This operation will delete files on your computer and in your org. Are you sure you want to proceed?',
```

Comment:
This text is used to ask the user for confirmation that they want to delete source
<img width="486" alt="screen shot 2018-08-28 at 1 48 50 pm" src="https://user-images.githubusercontent.com/1639375/44746959-23c40f00-aac9-11e8-8e9f-a56fc993eed1.png">

Reply:
That sounds great, I'll make that change now.